cli/script/command-executor.ts

```diff
@@ -166,14 +171,25 @@ function appAdd(command: cli.IAppAddCommand): Promise<void> {
         });
 }
 
+function getOwnerEmail(map: CollaboratorMap): string {
+    var ownerEmail: string = "";
+    if (map) {
+        ownerEmail = Object.keys(map).filter((email: string) => {
+            return map[email].permission === Permissions.Owner;
+        })[0];
```

Comment:
Yes, I missed that we already have getOwnerEmailFromCollaboratorMap(...) function, I will reuse the same, and rename it to getOwnerEmail(...)